libraries/chain/vesting_balance_object.cpp

```diff
@@ -117,7 +117,7 @@ asset cdd_vesting_policy::get_allowed_withdraw(const vesting_policy_context& ctx
       return asset(0, ctx.balance.asset_id);
    fc::uint128_t cs_earned = compute_coin_seconds_earned(ctx);
    fc::uint128_t withdraw_available = cs_earned / std::max(vesting_seconds, 1u);
-   assert(withdraw_available <= ctx.balance.amount.value);
+   assert(withdraw_available <= static_cast<fc::uint128_t>(ctx.balance.amount.value));
```

Comment:
This `assert` only works for debug build. I guess we want to remove it or change it to `FC_ASSERT`, but it belongs to a new PR (see https://github.com/bitshares/bitshares-core/issues/511).